themes/ra-theme/assets/css/quote-box.scss

```diff
@@ -1,19 +1,30 @@
 div.quote-box {
   display: grid;
   grid-auto-flow: column;
-  grid-auto-columns: 1fr;
+  grid-auto-columns: initial;
   column-gap: 2rem;
   align-items: stretch;
   overflow: hidden;
   border-radius: 6px;
-
   font-size: 12px;
   line-height: 1.3em;
-
-  padding: 1rem 2rem;
-  width: 400px;
-  height: 200px;
-
+  padding: 2rem 2rem;
+  width: 600px;
+  height: 100%;
+  @media (max-width: $break-width) {
+    display: grid;
+    grid-auto-flow: row;
+    grid-auto-columns: initial;
+    column-gap: 1rem;
+    align-items: stretch;
+    overflow: hidden;
+    border-radius: 6px;
```

Comment:
Please don't set properties in media queries that are identical to their default settings.

Reply:
Oh ok.. I will correct this I make the clean up shortly